src/services/patients.service.js

```diff
@@ -12,3 +13,16 @@ export const getDistrictWiseData = () => {
 export const getStats = () => {
   return axios.get(`${api}/stats`);
 };
+
+export const getTestingData = async () => {
+  try {
+    const resp = await axios.get('https://api.covid19india.org/state_test_data.json')
+    const result = {}
+    for (const stateDetail of resp.data.states_tested_data) {
+      if (stateDetail.totaltested) result[stateCodes[stateDetail.state].toLowerCase()] = stateDetail
+    }
+    return result
+  } catch(err) {
+    console.log(err)
+  }
+}
```

Comment:
I will expose an API from backend so that this doesn't have to be done on front-end. If you want I can add you in the backend repo. It is an express application.

Reply:
That would be great :)

I did check for the backend repo but I guess you have made it private. So went ahead and implemented the logic there.